ocw/lib/eks.py

```diff
@@ -105,40 +105,54 @@ def wait_for_empty_nodegroup_list(self, region: str, cluster_name: str, timeout_
         if self.dry_run:
             self.log_info("Skip waiting due to dry-run mode")
             return None
-        self.log_dbg("Waiting empty nodegroup list in {}", cluster_name)
+        self.log_dbg(f"Waiting empty nodegroup list in {cluster_name}")
```

Comment:
interesting , this means that when test is running log level is not DEBUG ? :thinking:  Maybe we can tweak log level and increase coverage ? :D